client/tests/node_api.rs

```diff
@@ -16,9 +16,9 @@ use iota_types::block::{
 };
 
 const DEFAULT_DEVNET_NODE_URL: &str = "http://localhost:14265";
-const DEFAULT_DEVNET_FAUCET_URL: &str = "http://localhost:14265";
```

Comment:
should we move all these to common/mod.rs?

Reply:
We probably need to do the same as in the wallet, but not in this PR